doc/classes/ProgressBar.xml

```diff
@@ -9,10 +9,27 @@
 	<tutorials>
 	</tutorials>
 	<members>
+		<member name="fill_mode" type="int" setter="set_fill_mode" getter="get_fill_mode" default="0">
+			The fill direction. See [enum FillMode] for possible values.
+		</member>
 		<member name="percent_visible" type="bool" setter="set_percent_visible" getter="is_percent_visible" default="true">
 			If [code]true[/code], the fill percentage is displayed on the bar.
 		</member>
 	</members>
+	<constants>
+		<constant name="FILL_BEGIN_TO_END" value="0" enum="FillMode">
+			The progress fills from begin to end horizontally.
```

Comment:
Maybe we could precise what being and end means? Or at least mention it's dependent on the language direction? Something like:
> The progress bar fills from begin to end horizontally, according to the language direction.

Reply:
Maybe:

> The progress bar fills from begin to end horizontally, according to the language direction. If [member Control.is_layout_rtl] is [code]false[/code], it fills from left to right, and if it is [code]true[/code], it fills from right to left.

Reply:
Is `"<method> is <return value>"` correct though? I think it should be `"<method> returns <return value>"`.

>The progress bar fills from begin to end horizontally, according to the layout direction. If [method Control.is_layout_rtl] **returns** [code]false[/code], it fills from left to right, and if it **returns** [code]true[/code], it fills from right to left.

Reply:
Yeah that's better. I thought initially that it's a property but it's indeed a method.